pr_review_queue.py

```diff
@@ -412,10 +422,15 @@ def main():
     parser.add_argument("--repo", help="Set a repo in `--org` on github.com", required=False)
     parser.add_argument("--queue", help="Create a review queue", default=True,
                         action=argparse.BooleanOptionalAction)
+    parser.add_argument("--slack-format", help="Generate slack format, otherwise use markdown", default=True,
```

Comment:
Default True - are you ok with it? You would need to change your script with `--slack-format`.

Reply:
I'm ok with that!
If it's not required, would the script really have to be amended?

If so, let's update it in the same PR: https://github.com/osbuild/pr-review-queue/blob/main/.github/workflows/pr-review-queue.yaml#L22

Reply:
Done